MC-33183: New In-store delivery feature. Updated existing information and added new topics. (#604)
magento/merchdocs: src/sales/checkout-step1-shipping.md

title: 'Checkout Step 1: Shipping'
---

> The first step of the checkout process is for the customer to complete the shipping address information, and to choose the delivery method. For more information, see [delivery methods]({% link configuration/sales/delivery-methods.md %}).

Reviewer: ```suggestion
The first step of the checkout process is for the customer to complete the shipping address information and choose the delivery method. For more information, see [delivery methods]({% link configuration/sales/delivery-methods.md %}).
```

Author: done!

![]({% link images/images/storefront-checkout-step1-shipping.png %}){: .zoom}
_Checkout Step 1: Shipping_

## Select In-store Delivery

> A new tab appears during checkout if the in-store delivery is enabled.

Author: Added, thanks!

![]({% link images/images/select-in-store-delivery.png %}){: .zoom}
_Checkout: Shipping Method_

> This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. Click **Select other** and a list of the available pickup locations appear.

Reviewer: ```suggestion
This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. When the customers clicks **Select other**, a list of the available pickup locations appears.
```

Author: thanks a lot!

> This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. Click **Select other** and a list of the available pickup locations appear.

Author: reviewed!

![]({% link images/images/select-in-store-delivery-populated.png %}){: .zoom}
_Checkout: Default Pickup Location_

> For more information about how to enable this shipping method, see [In-store Delivery]({% link shipping/shipping-in-store-delivery.md %}) feature.

Reviewer: ```suggestion
For more information about how to enable this method, see [In-store Delivery]({% link shipping/shipping-in-store-delivery.md %}).
```

Author: done
